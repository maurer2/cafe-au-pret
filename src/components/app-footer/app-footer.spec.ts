import { shallowMount } from '@vue/test-utils'
import AppFooter from './app-footer.vue'

describe('AppFooter', () => {
  let cmp: any

  it('renders', () => {
    cmp = shallowMount(AppFooter, {
    })
    expect(cmp.text()).toBe('AppFooter')
  })

  test('matches snapshot', () => {
    expect(cmp).toMatchSnapshot()
  })
})
