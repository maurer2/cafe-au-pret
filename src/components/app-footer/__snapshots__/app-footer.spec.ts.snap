// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter matches snapshot 1`] = `
<footer
  class="footer"
>
   AppFooter 
</footer>
`;
